app/src/forms/file/middleware/filePermissions.js

```diff
@@ -13,7 +13,8 @@ const currentFileRecord = async (req, res, next) => {
   let fileRecord = undefined;
   try {
     // Check if authed, can expand for API key access if needed
-    if (req.params.id && req.currentUser) {
+    if (req.params.id && (req.currentUser || req.apiUser)) {
+      // expanded for api user
```

Comment:
Can you please add tests in `app/tests/unit/forms/file/middleware/filePermissions.spec.js`? After running the tests you can check coverage in `app/coverage/lcov-report/index.html`

Reply:
I've added new tests for these. `filePermissions.js` and `apiAccess.js` are fully covered in the coverage report!